audonnx/core/model.py

```diff
@@ -49,10 +49,21 @@ class Model(audobject.Object):
         transform: callable object or a dictionary of callable objects
         device: set device
             (``'cpu'``, ``'cuda'``, or ``'cuda:<id>'``)
-            or a (list of) provider(s)_
+            or a (list of) `provider(s)`_
         num_workers: number of threads for running
             onnxruntime inference on cpu.
-            If ``None`` onnxruntime chooses the number of threads
+            If ``None`` and ``session_options`` is ``None``,
+            onnxruntime chooses the number of threads
+        session_options: :class:`onnxruntime.SessionOptions`
+            to use for inference.
+            If ``None`` the default options are used and the number of
+            threads for running inference on cpu is determined
+            by ``num_workers``. Otherwise, the provided options are used
+            and the ``session_options`` properties
+            :attr:`~onnxruntime.SessionOptions.inter_op_num_threads`
+            and :attr:`~onnxruntime.SessionOptions.intra_op_num_threads`
+            determine the number of threads
+            for inference on cpu and ``num_workers`` is ignored
```

Comment:
```suggestion
            If ``None`` the default options are used
            and the number of threads
            for running inference on cpu
            is determined by ``num_workers``.
            Otherwise,
            the provided options are used
            and the ``session_options`` properties
            :attr:`~onnxruntime.SessionOptions.inter_op_num_threads`
            and :attr:`~onnxruntime.SessionOptions.intra_op_num_threads`
            determine the number of threads
            for inference on cpu
            and ``num_workers`` is ignored
```